pnpm-lock.yaml

```diff
@@ -17045,7 +17025,7 @@ packages:
   /@fluidframework/protocol-base/2.0.2:
     resolution: {integrity: sha512-9AbQ8FuZ5UNZgOWmNaWxs7CH08H9qRYX6OkTUbRXNjdJaFRSCBCkPSA23aHD85EN49UEkkpfoyH1SsQchHuR4Q==}
     dependencies:
-      '@fluidframework/common-utils': 3.0.0
+      '@fluidframework/common-utils': 3.1.0
```

Comment:
Not sure what's in this bump, but would be better to avoid it I think.  Could be ok though since it's minor.

Reply:
Like maybe just manually go revert these lines

Reply:
Wouldn't the 3.1 version still be installed for anyone consuming this release though regardless of the lock file?